collections/card4l.yaml

```diff
@@ -0,0 +1,122 @@
+Name: Sentinel-1 Radiometric Terrain Corrected SAR Backscatter
+Description: |
+ Sentinel-1 Radiometric Terrain Corrected SAR Backscatter is a product processed from [Sentinel-1 ground range detected (GRD)](https://docs.sentinel-hub.com/api/latest/data/sentinel-1-grd/) and is compliant with [CEOS Analysis Ready Data for Land (CARD4L)](https://ceos.org/ard/) specifications for Normalised Radar Backscatter (NRB) products. CARD4L compliant products fulfil requirements for general metadata, per-pixel metadata, radiometric and atmospheric corrections and geometric corrections outlined in the NRB [product family specifications](https://ceos.org/ard/files/PFS/NRB/v5.5/CARD4L-PFS_NRB_v5.5.pdf).
+ These products are processed using Sentinel Hub [CARD4L generation tool](https://apps.sentinel-hub.com/s1-card4l/) and are radiometrically calibrated to γ° (gamma-nought) backscatter, and provided as linear backscatter power. Full radiometric terrain correction (RTC) based on the technique developed by David Small[1], has been applied using the [Copernicus DEM](https://docs.sentinel-hub.com/api/latest/data/dem/) in order to mitigate topographic terrain effects on the backscatter induced by side-looking geometry of SAR imaging and compounded by rugged terrain. Speckle filtering has not been applied in order to preserve spatial resolution and user freedom. 
+ The twin Sentinel-1A and 1B satellites are designed to acquire data with a revisit cycle of 6 to 12 days providing continuous all-weather, day and night imagery for Land and Maritime Monitoring. Archived Sentinel-1 CARD4L NRB data VV and VH polarization over Africa are provided, with new acquisitions being updated daily as they become available. To obtain Sentinel-1 CARD4L NRB data for other parts of the world, registered users have the possibility of custom processing using [CARD4L generation tool](https://apps.sentinel-hub.com/s1-card4l/). 
```

Comment:
I feel like something is missing between `data` and `VV` in the sentence 
`Archived Sentinel-1 CARD4L NRB data VV and VH polarization over Africa are provided`

Maybe `in` - with it it sounds similar to the descriptions of bands under `BandInformation`

Reply:
I see it too, I changed it ,  I believe it is more clearer now
`This collection is comprised of CARD4L compliant Sentinel-1 VV and VH backscatter products over Africa for archived data, with new acquisitions being updated daily as they become available. `

Reply:
Thanks, this is great now.